### DISCUSSION TITLE
More failing to download YouTube videos

### DISCUSSION THREAD

**Discussion by FarneyBlather** [aclap-dev/video-downloadhelper#1409] [Help and Q&A]
### Your question

I had previously found a couple of related YouTube videos that could not be downloaded—two MSNBC sports. 

I ran into another YouTube video that will not download, but from a different source. What's more, a pattern is coming into focus.

https://www.youtube.com/watch?v=HzFjwC1p9EY&t=1s

Yesterday, I could not download the video. Today, I can (in fact, I just finished downloading it).

Yesterday, restarting the browser did not help. Yesterday, there was NO way to download the video.

Yesterday, I watched the video a couple times. I moved the playback slider all over the place. I may have accidentally closed the tab and reopened it. Point is, it appears that "messing with" a video before downloading it may trip up detection. That is what happens, after all—the addon fails to detect available video at times. This may be related to it occasionally mistitling videos: attaching the title of a previous video to the video currently playing and being downloaded, or titling a video "YOUTUBE - YOUTUBE," rather than the actual title.

Yes, in both earlier failures to download a YouTube video, I has messing with playback or refreshing tabs.

### Addon & CoApp version and details

version: 9.1.0.44
target: mozilla
channel: stable
lang: en-US
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"accepted":true,"email":"<removed>","key":"4a6b7781ef93409f"}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:131.0) Gecko/20100101 Firefox/131.0
{
    "valid_license_message_has_been_displayed": true,
    "download_directory": "E:\\Videos",
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "open_count_store": 2,
    "successfull_dl": 88,
    "history_limit_in_days": 1
}

**Comment by markusbiro** [comment]
Hello!

I have the same observation as [FarneyBlather](https://github.com/FarneyBlather):
-some videos fails to download
-some downloads are mistitled/named to the previous video
-refresh/restart of the browser/system doesn't solve the issue.

If you have any idea/hint how can this be solved, please let me know!

Thank you in advance!
Emeric
